melted_UKBB_extract.py

```diff
@@ -329,16 +335,7 @@ def extract_UKBB_tabular_data(
         ],
     )
 
-    try:
-        with open(args.config_file, "r") as stream:
-            try:
-                config = yaml.safe_load(stream)
-            except yaml.YAMLError as exc:
-                logging.exception(exc)
-                sys.exit(1)
-    except FileNotFoundError as exc:
-        logging.exception(exc)
-        sys.exit(1)
+    config = Config.from_yaml(args.config_file)
```

Comment:
Switching to this implementation means you can remove the `import yaml` right?

Reply:
Yes, that is correct

Reply:
c71b2fe2519f596f0abf25b3bfe990e03f0bb075